app/forms/bulk_action_form.rb

```diff
@@ -48,4 +55,12 @@ def pids_with_prefix
 
     pids.split.flatten.map { |pid| pid.start_with?('druid:') ? pid : "druid:#{pid}" }.join("\n")
   end
+
+  def options_for_use_license_type
+    Constants::LICENSE_OPTIONS.map do |key, attributes|
+      next if attributes.key?(:deprecation_warning)
+
+      [attributes.fetch(:label), key]
```

Comment:
can we use `attributes[:uri]` here rather than `key`?  That gets us closer to using the 4.0 licenses.

Reply:
this too?

Reply:
will do